File: app/src/main/java/com/theworld/vibratestrings/ui/ChatActivity.java
package com.theworld.vibratestrings.ui;

import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

import com.theworld.vibratestrings.R;

public class ChatActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_chat);
    }


    public void click(View view) {
        String url = "https://chat.whatsapp.com/GsLd9XUEdHFEa5E577LevA";
        Intent link = new Intent(Intent.ACTION_VIEW, Uri.parse(url));
        startActivity(link);
    }
}

File: app/src/main/java/com/theworld/vibratestrings/ui/ChatActivity.kt
package com.theworld.vibratestrings.ui

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.theworld.vibratestrings.R
import android.content.Intent
import android.net.Uri
import android.view.View
import coil.load
import com.theworld.vibratestrings.databinding.ActivityChatBinding
import com.theworld.vibratestrings.databinding.ActivityDashboardBinding

class ChatActivity : AppCompatActivity() {

    private lateinit var binding: ActivityChatBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivityChatBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.imageView14.load(R.drawable.chatroom)

    }

    fun click(view: View?) {
        val url = "https://chat.whatsapp.com/GsLd9XUEdHFEa5E577LevA"
        val link = Intent(Intent.ACTION_VIEW, Uri.parse(url))
        startActivity(link)
    }
}